docs/50-sdk.md

```diff
@@ -223,10 +224,204 @@ Find some samples and getting stated tutorial in the `examples/sdk/` directory o
         - str
 
 
+- `submit_job(self, job_configuration)`
+
+    Submit an AZTK Spark Job
+
+    Parameters:
+
+        - job_configuration: aztk.spark.models.JobConfiguration
+            The configuration of the job to be submitted
+    
+    Returns:
+        
+        - None
+
+- `list_jobs(self)`
+
+    List all created AZTK Spark Jobs
+
+    Parameters:
+
+        - job_configuration: aztk.spark.models.JobConfiguration
+            The configuration of the job to be submitted
+    
+    Returns:
+        
+        - List[aztk.spark.models.Job]
+       
+- `list_applicaitons(self, job_id)`
+
+    List all applications created on the AZTK Spark Job with id job_id
+
+    Parameters:
+
+        - job_id: str
+            The id of the Job
+    
+    Returns:
+        
+        - List[aztk.spark.models.Application]
+       
+- `get_job(self, job_id)`
```

Comment:
Similar to comment above... We created a schedule, but maybe have no way to reference it or modify it.

Reply:
That's by design for now. At this stage, for job submission, there isn't really a schedule. (technically there is under the hood, but it is not exposed to the user yet). So it doesn't really make sense to have an update to an AZTK Job, becuase AZTK jobs run exactly once the instant they are created.

Once we have scheduling we should definitely add an update/modify AZTK job method.